pkg/sql/pgwire/testdata/pgtest/row_description

```diff
@@ -194,23 +194,21 @@ BindComplete
 {"Type":"ParseComplete"}
 {"Type":"BindComplete"}
 
-# The following discrepancy is another bug.
-# See: https://github.com/cockroachdb/cockroach/issues/49215
 until noncrdb_only ignore_table_oids
 RowDescription
 ----
-{"Type":"RowDescription","Fields":[{"Name":"b","TableOID":0,"TableAttributeNumber":2,"DataTypeOID":1042,"DataTypeSize":-1,"TypeModifier":12,"Format":0}]}
+{"Type":"RowDescription","Fields":[{"Name":"b","TableOID":0,"TableAttributeNumber":2,"DataTypeOID":1042,"DataTypeSize":-1,"TypeModifier":12,"Format":0},{"Name":"?column?","TableOID":0,"TableAttributeNumber":0,"DataTypeOID":23,"DataTypeSize":4,"TypeModifier":-1,"Format":0}]}
```

Comment:
Hmm, how come TableOID is 0 here then? I would expect it to be the oid of the table `tab3`

Reply:
when the `ignore_table_oids` flag is set that field is set to zero -- its for comparing easily to the postgres output (which is why the `nocrdb_only` flag is also set.

Reply:
Heh, so then is there an actual test of the new behavior anywhere?

Reply:
Its right below this!

Reply:
Oh, sorry... I saw the `TableOID:0` entry on the same line.